bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ScanAdapter.java

```diff
@@ -90,13 +90,22 @@ public RowFilter buildFilter(Scan scan, ReadHooks hooks) {
   @Override
   public Builder adapt(Scan scan, ReadHooks readHooks) {
     throwIfUnsupportedScan(scan);
+    byte[] startRowBytes = scan.getStartRow();
+    byte[] stopRowBytes = scan.getStopRow();
+
+    ByteString startRow = ByteStringer.wrap(startRowBytes);
+    // For gets, startRow == stopRow.  There's no need to create a new ByteString for stopRow
+    ByteString stopRow =
+        startRowBytes == stopRowBytes ? startRow : ByteStringer.wrap(stopRowBytes);
```

Comment:
reference is important here... Get operations use the same reference for start/stop.  If we do comparisons, we hit a performance issue, so I'd rather not do that.  I'll document the intent.